### PR TITLE
Hometask 6

### PR DESCRIPTION
**Pull request by alexandrali**

### CODE REVIEW

File: src/loaders/express.ts [hunk 1]
```diff
@@ -70,6 +74,9 @@ export default ({app}: {app: express.Application}) => {
       error: RESPONSE_MESSAGES.INVALID_URL,
     });
     switch (err.message) {
+      case AUTH_MESSAGES.INVALID_CREDENTIALS:
```
Comment (titivuk) [inline src/loaders/express.ts:77]: what about NO_TOKEN?

File: src/services/auth.service.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import {Users} from '../loaders/database';
+import bcrypt from 'bcrypt';
+import {AUTH_MESSAGES} from '../config/messages';
+
+export default {
+  async login(login: string, password: string) {
```
Comment (titivuk) [inline src/services/auth.service.ts:8]: I think you can create JWT token here and remove this logic from controller layer

File: src/services/auth.service.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      where: {
+        login: login,
+      },
+    });
+
+    if (!user) {
+      throw new Error(AUTH_MESSAGES.INVALID_CREDENTIALS);
+    }
+
+    const passwordMatches = bcrypt.compareSync(
```
Comment (titivuk) [inline src/services/auth.service.ts]: it's better to use async version

File: src/loaders/express.ts (hunk 1 above)
Comment (alexandrali) [reply src/loaders/express.ts:77]: added

File: src/services/auth.service.ts (hunk 2 above)
Comment (alexandrali) [reply src/services/auth.service.ts:8]: fixed

File: src/services/auth.service.ts (hunk 3 above)
Comment (alexandrali) [reply src/services/auth.service.ts]: fixed

**Comment by titivuk** [comment]
LGTM

**Pull request merged**